studio-ui/ui/scss/src/global.scss

```
@@ -584,7 +584,7 @@ div.ttTableGroup {
 	text-align: left;
 	vertical-align: middle;
 	white-space: wrap;
-	word-wrapord-wrap: break-word;
+	word-wrap: break-word;
```

Stylelint (17.14.1)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)